src/rmp/include/rmp/blif.h

```diff
@@ -90,7 +91,7 @@ class Blif
   std::string const1_cell_port_;
   std::map<std::string, std::pair<float, float>> requireds_;
   std::map<std::string, std::pair<float, float>> arrivals_;
-  static int call_id_;
+  static std::atomic<int> call_id_;
```

Comment:
@QuantamHD I see no reason for this to be static at all.  Can we just make it a class member?